lib/scrub/max_ocn.rb

```diff
@@ -66,7 +66,7 @@ def make_call
       if @mock
         IO.read("spec/fixtures/max_oclc_response.json")
       else
-        URI.parse(OCLC_URL).open.read
+        Faraday.get(OCLC_URL).body
```

Comment:
We're including faraday in the gemfile anyway (not 100% sure why), but we might as well use it in preference to open-uri, not sure it makes much difference really though.